### ISSUE TITLE
lamdu.org leads to warning and then to completely different website

### ISSUE THREAD

**Issue by janreppien** [lamdu/lamdu#231]
I stumbled about this project and the README does a petty good job of explaining the concepts. Looking for even more information I clicked the link shown in the sidebar on the right to [lambd.org](https://lamdu.org/). My browsers (tested with Firefox and Chromium) first displayed a Warning about a potential unsafe website and when I proceeded anyway I was led to this completely different website [chloehom.com](https://www.chloehom.com/). 

![image](https://github.com/user-attachments/assets/1f5ce0df-576e-4c6a-9859-c10dc23756ca)

**Comment by yairchu** [comment]
Wow, indeed going to https://lamdu.org does the same for me!
While going to https://www.lamdu.org does work so that's a workaround for now, also updated side-bar.

We use some "naked domain redirect" thing from nakedssl.com:

<img width="451" alt="image" src="https://github.com/user-attachments/assets/ea58698b-c453-4737-b461-34c297bb84ab">

Looks like I got an email from them back in November that lamdu.org was removed from my account. I suppose someone nefarious removed it for me and then added it to their account so it would redirect to that scam site.

Thanks for letting us know!
